components/clarity-repl/src/repl/interpreter.rs

```diff
@@ -1726,30 +1726,6 @@ mod tests {
         ));
     }
 
-    #[test]
```

Comment:
Moved to dedicateds integration tests directory `./clarity-repl/tests/boot_contracts.rs`